adapter/pkg/config/log_types.go

```diff
@@ -115,6 +123,12 @@ func getDefaultLogConfig() *LogConfig {
 				"extAuthDtls":   "%DYNAMIC_METADATA(envoy.filters.http.ext_authz:extAuthDetails)%",
 			},
 		},
+		InsightsLogs: &insightsLog{
+			Enable:          false,
+			LogFile:         "/dev/stdout",
+			LoggingFormat:   "-",
```

Comment:
Does it work if we simply provided like this?

Reply:
even this is enabled it will just print '-'. but ideally it is not usable and proper config should apply